### DISCUSSION TITLE
Homebridge support for HomeKit secure video released

### DISCUSSION THREAD

**Discussion by steverae68** [Sunoo/homebridge-camera-ffmpeg#1256] [Ideas]
Need I say more....

**Comment by Sunoo** [comment]
https://github.com/Sunoo/homebridge-camera-ffmpeg/issues/338

I am working on it. I had a pipe burst recently so that was a bit of a set back, but I am still hoping to have the feature added soon.

**Reply by kungktd** [reply]
Sorry to hear about your busted pipe but looking forward to HSV.

**Reply by steverae68** [reply]
Any idea when we can see this implemented in a beta...

I switched over to cameraUI to test the implementation out and have to say, I'm very impressed indeed.

CameraUI adds a lot more things I don't need so would love to get this enabled on my good old trusty ffmpeg plugin I've been using for years....

I'm no plumber but anything I can do to help, just reach out

**Reply by steverae68** [reply]
Here is some info that might help

(https://github.com/Supereg/secure-video-specification)

**Reply by Sunoo** [reply]
With any luck, beta some time this weekend, but no promises.

**Reply by riiveraluis** [reply]
@Sunoo Excited to soon use HKSV. I see your branch of it, has the necessary components to also have prebuffering I wonder what raspberry pi will support it. Currently I had no luck with scrypted on a Zero 2 only using streaming while your plugin runs just fine with streaming. Best of luck and thanks for your work. Hope to see a beta really soon.

**Reply by Sunoo** [reply]
@riiveraluis Zero 2 likely could handle it, provided you don’t have too many cameras. I haven’t been able to get my hands on a Zero 2 yet myself, so I can’t say for sure.

**Reply by riiveraluis** [reply]
Ready to test when you are. I have 4 cameras running with dahua alert plug-in.

**Reply by riiveraluis** [reply]
Do you have an updated ETA for HKSV?

**Reply by Sunoo** [reply]
@riiveraluis As soon as I have enough mental energy left after wrangling my day job, insurance adjuster, and contractors. There really shouldn’t be too many development hours left, the trick is just having my head in the right place to be able to focus on it. Things seem to be settling down a bit finally though, so hopefully soon.

**Reply by steverae68** [reply]
I've donated to you, hope you get a few coffee's with my donation to help you focus 👍

**Comment by steverae68** [comment]
Oh that’s good to know… not your pipe burst, but the fact this feature is coming soon.   

I’ve been using your plugin for ALL my camera’s for Donkeys years and it’s never let me down, HSV is the icing on the cake we’ve all been waiting for 

Thank you.

If you need any testing, happy to help

**Comment by steverae68** [comment]
Same here, with 3 hikvision camera’s.  Can’t wait to get back to this plugin… I use the smtp motion detection add on, works a treat…. I’m just amazed at how responsive the cameras are through HKSV 

I use a pi 3b so  issues with with performance 

Recommend releasing in beta first

**Comment by steverae68** [comment]
Any further update on this ?

**Comment by steverae68** [comment]
@riveralius firstly, I’m not desperate, I’ve been using HKSV via CameraUI now for ever a month without any issues… I’m only curious about any updates as the last we heard from the dev was over 2 weeks ago, where they mentioned a possible beta that weekend.

I get people have lives and other commitments, just keep us in the loop that is all.

Even if they said, ‘won’t be done for 6 months…’ then we wouldn’t wonder for 6 months …..

It’s simply a case of setting expectations that is all…

**Reply by Sunoo** [reply]
It should be on the scale of days to possibly weeks. It should not be months.

**Reply by P13ter** [reply]
@Sunoo Any progress with adding the secure video support?

**Reply by Sunoo** [reply]
My house is finally almost back together, so I should have fewer things demanding my attention. It has been close for a while now.

**Reply by mlamoure** [reply]
Any update you can provide on the state of hksv?

**Comment by bradleyhodges** [comment]
Any update on HSV support? @Sunoo
